### DISCUSSION TITLE
Node-RED automation stopped working (Occupancy events)

### DISCUSSION THREAD

**Discussion by szaske** [blakeblackshear/frigate#4386] [General]
I noticed that my Home Assistant Automation, which used Node-RED has stopped working. I believe it was because this newest version of Frigate changed the name of sensors that appear in Home Assistant. The current broken automation is attempting to connect to an entity called `sensor.driveway_zone_person`. However I no longer see an entity with this name in Home Assistant. This automation would fire when a person went into the zone called "driveway-zone".  The closest thing I see now is an entity called `binary_sensor.driveway_zone_person_occupancy`.  Can anyone explain if this is the same event? 

I'm looking for the event that fires when a person enters my driveway zone and ends when they leave the driveway zone.  I'm hoping this event has just been renamed.

**Comment by NickM-27** [comment]
> I'm looking for the event that fires when a person enters my driveway zone and ends when they leave the driveway zone. I'm hoping this event has just been renamed.

`binary_sensor.driveway_zone_person_occupancy` would be that
